### ISSUE TITLE
Cannot able to print trace level logs

### ISSUE THREAD

**Issue by h4sh3d** [farcaster-project/farcaster-node#235]
With verbose flag set to `-vvvv` trace level doesn't show up, INFO and DEBUG are but not TRACE.

**Comment by zkao** [comment]
I can't reproduce

**Comment by h4sh3d** [comment]
Yeah it's weird behavior on my side. Not super important for now, I'll investigate further.

**Comment by TheCharlatan** [comment]
Closing this, since the behavior seems to have been fixed.
